src/ui/preview/Playground.tsx

```diff
@@ -68,7 +86,10 @@ async function initPlayground(
 	};
 
 	const client: PlaygroundClient = await startPlaygroundWeb( options );
-	await client.isReady;
+	await client.isReady();
+
+	localStorage.setItem( 'isWordPressInstalled', 'true' );
```

Comment:
I think we should namespace this with the slug, because otherwise this value will be shared across sessions (aka sites).
```suggestion
	localStorage.setItem( `${slug}-isWordPressInstalled`, 'true' );
```